Kwf/Form/Field/MultiCheckbox.php

```diff
@@ -207,7 +207,7 @@ protected function _getFields()
 
     public function hasChildren()
     {
-        return count($this->_fields) > 0;
+        return ($this->_fields instanceof Countable) ? count($this->_fields) > 0 : false;
```

Comment:
```suggestion
        return (is_array($this->_fields) || $this->_fields instanceof Countable) ? count($this->_fields) > 0 : false;
```

Reply:
Falls $this->_fields ein array sein kann muss man is_array() explicit prüfen,
array implementiert Countable nämlich nicht

```php
$array = array();
var_dump($array instanceof Countable);
// bool(false)

```

Reply:
ich glaub $this->_fields kann nur vom Typ Kwf_Collection_FormFields sein oder?

Reply:
Dann passts!